packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/quanticGeneratedAnswer.e2e.ts

```diff
@@ -232,6 +233,33 @@ useCaseTestCases.forEach((useCase) => {
                 expect(showMoreButtonLabel).toEqual(expectedShowMoreLabel);
               });
             });
+
+            // This test is to make sure that a generate call is made when selecting a facet with the answer api enabled
+            if (config.options.answerConfigurationId) {
+              test.describe('when selecting a facet after the answer is generated', () => {
+                test.use({
+                  options: {
+                    ...config.options,
+                    withFacets: true,
+                  },
+                });
+                test('should trigger a new generate call to the answer API', async ({
+                  generatedAnswer,
+                }) => {
+                  await generatedAnswer.streamEndAnalyticRequestPromise;
+
+                  const generateRequestPromise =
+                    generatedAnswer.waitForGenerateSubmitRequest({
+                      searchHub:
+                        useCase.value === 'insight'
+                          ? 'UI-KIT Insight Panel'
+                          : 'default',
```

Comment:
The test should not depend on a search hub value tied to a given answer configuration id.
why checking for the search hub here?

This is a bad pattern that we should avoid because the search hub value can easily be changed in the platform breaking this test and the CI in UI-KIT.

Reply:
yeah good point, will remove